Makefile

```diff
@@ -6,9 +6,7 @@ endif
 
 .PHONY: run
 run:
-	export FLASK_APP=app.py; \
-	export FLASK_DEBUG=1; \
-	flask run --port $(FLASK_PORT) --host $(FLASK_HOST)
```

Comment:
The [Flask CLI](http://flask.pocoo.org/docs/0.12/server/#command-line) automatically reloads the application when making source file changes which is convenient for development. It would be preferable to continue using this if possible.